reana-ui/src/pages/workflowList/components/WorkflowActionsPopup.js

```diff
@@ -0,0 +1,77 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import { Icon, Menu, Popup } from "semantic-ui-react";
+import { useDispatch } from "react-redux";
+
+import { deleteWorkflow } from "../../../actions";
+
+export default function WorkflowActionsPopup({
+  workflow,
+  setOpenDeleteModal,
+  setSelectedWorkflow,
+}) {
+  const dispatch = useDispatch();
+  const [open, setOpen] = useState(false);
+  const { id, name, run, size, status } = workflow;
+  let menuItems = [];
+  if (status === "deleted") {
```

Comment:
Some crazy idea - maybe we need to introduce another status for deleted workflows with deleted workspaces?
If it's not needed anywhere else (in the backend), we could simply set it in the redux reducers. It would simplify workflow list rendering as well

Reply:
I'm not sure, while it might help in these specific cases you mentioned, it can be tricky for other (if not implemented server-side) such as filtering. Let's see what others think @tiborsimko @diegodelemos @ParthS007

Reply:
Yeah, I think different `status` for `workflow` and `workspace` sounds good. As we develop further, it will be useful for having different `status` for both. But we have to add in the server-side too if we decide to separate both.

Reply:
I think we can live with this for the time being as the logic is not so complex... introducing an "inconsistency" with the backend might bite us in the future. We could make it more idiomatic if we have a new property on worfklows which can be `isWorkspaceEmpty` (or some other better name), at the end that is going to do the same check as we have now `size == "0K"`....